qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/create/CreateOfferInput.groovy

```diff
@@ -9,4 +15,21 @@ package life.qbic.portal.portlet.offers.create
  */
 interface CreateOfferInput {
 
+
+    /**
+     * Creates a new quotation based on the given TOMATOID
+     * @param tomatoId makes it possible to identify the quotation
+     * @since 1.0.0
+     */
+    void createOffer(String tomatoId)
+
+    /**
+     * Method to create a new offer for the provided offer information
+     * @param projectTitle describing the title of the project
+     * @param projectDescription a description of the project
+     * @param customer as the customer of the project
+     * @param projectManager as the manager of the project
+     * @param productItems which are requested by the customer
+     */
+    void createNewOffer(String projectTitle, String projectDescription, Person customer, Person projectManager, List<ProductItem> productItems)
```

Comment:
Does this method save the newly created offer also?

```suggestion
    void createNewOffer(String projectTitle, String projectDescription, Person customer, Person projectManager, List<ProductItem> productItems)
```

Reply:
This will be done after the offer is created but basically triggered after its creation from this method

Reply:
Then this method is an internal one and part of a createAndSave method and should be private.
```suggestion
    private Offer createNewOffer(String projectTitle, String projectDescription, Person customer, Person projectManager, List<ProductItem> productItems)
```